app/templates/public/index.hbs

```diff
@@ -12,4 +12,10 @@
       {{public/speaker-list speakers=nonFeaturedSpeakers}}
     </div>
   </div>
+  <br>
+  <div class="sponsors">
+    <div class="row">
+      {{public/sponsor-list sponsors=sponsorsByCategory}}
```

Comment:
Let the sponsors grouping be done by the sponsor-list component. This is the components job. You just should pass the plain sponsors array to it.